File: programme/hours-to-seconds/hours-to-seconds.c
// Write a C programme to convert hours to seconds.

#include <stdio.h>

int main()
{
	double input;
	printf("Hours: ");
	scanf("%lf", &input); 
	int seconds = input * 3600; // seconds conversion

    printf("Seconds: %d\n", seconds);
    return 0;
} 

File: programme/hours-to-seconds/hours-to-seconds.cs
// Write a programme to convert hours to seconds.
using System;

public class HoursToSeconds
{
    public static void Main(string[] args)
    {
        Console.Write("Hours : ");
        int num;
        if (int.TryParse(Console.ReadLine(), out num))
        {
            Console.WriteLine("\nSeconds : {0}", num * 3600);
        }
        else
        {
            Console.WriteLine("Input Err");
        }
    }
}
